x-pack/plugin/esql/qa/testFixtures/src/main/resources/floats.csv-spec

```diff
@@ -592,3 +592,42 @@ s:double | emp_no:integer | salary:integer  | salary_change:double
 -1.0     | 10004          | 36174           | [-0.35, 1.13, 3.65, 13.48]
 -1.0     | 10005          | 63528           | [-2.14, 13.07]
 ;
+
+scalb
+required_capability: fn_scalb
+
+row x = 3.0, y = 10 | eval z = scalb(x, y);
+
+x:double | y:integer | z:double
+3.0      | 10        | 3072.0
+;
+
+scalbWithNonFoldableArgs
+required_capability: fn_scalb
+
+from employees
+| eval s = scalb(mv_min(salary_change), emp_no / 1000)
+| keep s
+| limit 3;
+
+s:double
+1218.56 
+-7403.52
+13127.68
+;
+
+scalbWithEvalAndSort
```

Comment:
Please add some tests with very large long scale factors, above max integer, both foldable and non-foldable